README.md

```diff
@@ -26,22 +26,23 @@ An array of patterns to ensure are present in the specified `.gitignore` file. A
 
 <a id="comment">
 
-#### comment (?string)
+#### comment (?string, default: 'managed by ensure-gitignore')
 
-Prepended to the list of provided pattens at the bottom of the file, to indicate which patterns are being controlled programmatically.
+Allows you to document the list of patterns that are being controlled programmatically. This helps provide context as to why/what is controlling certain patterns within a `.gitignore` file. Anything inside the comment block will be re-written on each run.
```

Comment:
Do we need to document that this option is required if you want to be able to remove controlled patterns? Otherwise, this looks like it's merely cosmetic.

Reply:
Given there is a default comment you get the removal behaviour out of the box rather than an opt-in. There is no way to opt-out of the block style.

Reply:
Ah, gotcha. So should we make it mandatory? Right now it's `?string`.

Reply:
Ignore me. I need more coffee.